src/main/cpp/t10_unique.cpp

```diff
@@ -21,9 +21,39 @@
 
 #include "t10_unique.h"
 #include <iostream>
-
+#include <vector>
 using namespace std;
 
 int t10_unique() {
+int n, temp, sum = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'sum' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4719943)